src/backend/app.js

```diff
@@ -20,12 +20,35 @@ app.use(express.json());
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+app.get('/my-route', (req, res) => {
+  res.send("Hi friend");
+});
 
 if (process.env.API_PATH) {
   app.use(process.env.API_PATH, router);
 } else {
   throw "API_PATH is not set. Remember to set it in your .env file"
 }
+app.get("/future-meals", async (req, res) => {
+  const rows = await knex.raw("SELECT * FROM `meal` WHERE `when` > NOW()");
```

Comment:
Future meals include the meals that we have today so, `when >= now()` and for Past meals `when < now()`.

Reply:
👍🏻